ydb/docs/en/core/toc_i.yaml

```diff
@@ -36,7 +36,9 @@ items:
     mode: link
     path: downloads/toc_p.yaml
 - name: Public talks
```

Comment:
articles aren't really public **talks**, let's rename this to something like "Public materials"

Reply:
Done